pkg/controller/storagecluster/cephcluster.go

```diff
@@ -475,3 +484,29 @@ func newCephDaemonResources(custom map[string]corev1.ResourceRequirements) map[s
 
 	return resources
 }
+
+func (r *ReconcileStorageCluster) throttleStorageDevices(storageClassName string) (bool, error) {
+	storageClass := &storagev1.StorageClass{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: "", Name: storageClassName}, storageClass)
+	if err != nil {
+		return false, fmt.Errorf("failed to retrieve StorageClass %q. %+v", storageClassName, err)
+	}
+	bool flag := false
+	switch storageClass.Provisioner {
+	case string(EBS):
+		if contains(throttleDiskTypes, storageClass.Parameters["type"]) {
+			flag = true
+			sc.Spec.StorageDeviceSets[i].Config.TuneDeviceClass = true
```

Comment:
I dont see any variable of SC where did you get it.

Reply:
```suggestion
			sc.Spec.StorageDeviceSets[i].Config.TuneFastDeviceClass = true
```